ledger-tool/src/main.rs

```diff
@@ -1016,6 +1021,29 @@ fn main() {
             .arg(&halt_at_slot_arg)
             .arg(&hard_forks_arg)
             .arg(&max_genesis_archive_unpacked_size_arg)
+            .arg(
+                Arg::with_name("warp_epoch")
+                    .required(false)
+                    .long("warp-epoch")
+                    .takes_value(true)
+                    .value_name("WARP_EPOCH")
+                    .help("After loading the snapshot epoch warp the ledger to EPOCH_SLOT, \
+                           which could be an epoch in a galaxy far far away"),
+            )
+            .arg(
+                Arg::with_name("force_inflation")
+                    .required(false)
+                    .long("force-inflation")
+                    .takes_value(false)
+                    .help("Force inflation"),
+            )
+            .arg(
+                Arg::with_name("reset_capitalization")
+                    .required(false)
+                    .long("reset-capitalization")
+                    .takes_value(false)
+                    .help("Reset capitalization"),
+            )
```

Comment:
`--recalculate-capitalization`?  When I saw "resetting", I first though that this would set the bank capitalization to 0